src/murmur/Meta.h

```diff
@@ -28,6 +29,13 @@ class MetaParams {
 public:
 	QDir qdBasePath;
 
+	QList< QSslError::SslError > qlAllowedSslClientErrors;
+	bool bMctsIncludeHostCAs;
+	bool bMctsIncludeOwnCAs;
+	bool bMctsIncludeOwnCert;
+	QList< QSslCertificate > qlMcts;
+	bool bForceUsernameCertSubjectEquality;
```

Comment:
```suggestion
	bool bUsernameCertCommonNameMatch;
```

Reply:
While I don't mind renaming this property, I don't like `bUsernameCertCommonNameMatch` as new name: In my opinion this sounds like "a fact", while it should sound more like a condition (or a command) that must be fulfilled (or checked) by the server to allow a user on the server. What about `bForceUsernameCertCommonNameMatch` or `bUsernameMustMatchCertCommonName`?

Reply:
I agree that this is lacking the sound of a condition. I second the proposed `bUsernameMustMatchCertCommonName`